backend/resources/hacknight.py

```diff
@@ -44,3 +44,12 @@ def post(self):
 
         return {'message': 'Hacknight created successfully.',
                 "hacknight": data}, HTTPStatus.CREATED
+
+
+class HacknightDetails(Resource):
+    def get(self, id):
+        hacknight_schema = HacknightSchema(only=('date', 'id', 'participants'))
```

Comment:
why do we limit fields using `only` here?

Reply:
This was intentional, as the ticket stated this (only these fields to be returned). Currently we don't have others, so could be removed. :-)

Reply:
I believe it should be removed. I don't even see the word `only` in issue description so would assume that was to point out all fields in that model.